src/common/fullscreenNavigation/FullscreenNavigation.tsx

```diff
@@ -42,17 +42,21 @@ function FullscreenNavigation(props: Props) {
           />
         </div>
         <div className={styles.navItems}>
-          <NavLink to="/" className={styles.navLink} onClick={closeNav}>
-            {t('nav.information')}
-          </NavLink>
-          <NavLink
-            to="/connected-services"
-            className={styles.navLink}
-            onClick={closeNav}
-          >
-            {t('nav.services')}
-          </NavLink>
           {isAuthenticated && (
+            <React.Fragment>
```

Comment:
Shouldn't our version of babel support `<>`? Do you simply prefer this notation?

Reply:
I just prefer it 😄

Reply:
I prefer <> - wonder if we're going to start a vi vs emacs war 😄